SPECS/kernel/config

```diff
@@ -526,7 +526,7 @@ CONFIG_CALL_PADDING=y
 CONFIG_HAVE_CALL_THUNKS=y
 CONFIG_CALL_THUNKS=y
 CONFIG_PREFIX_SYMBOLS=y
-CONFIG_SPECULATION_MITIGATIONS=y
+CONFIG_CPU_MITIGATIONS=y
```

Comment:
Result of commit: https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/8292f4f8dd1b005d0688d726261004f816ef730a